src/pages/location-message/location-message.ts

```diff
@@ -0,0 +1,66 @@
+import { Component, OnInit, OnDestroy } from '@angular/core';
+import { NavController, ViewController, AlertController, Platform } from 'ionic-angular';
+import { Geolocation } from 'ionic-native';
+import { Location } from 'api/models';
+
+const DEFAULT_ZOOM = 8;
+
+@Component({
+  selector: 'location-message',
+  templateUrl: 'location-message.html'
+})
+export class NewLocationMessageComponent implements OnInit, OnDestroy {
+  lat: number = 51.678418;
+  lng: number = 7.809007;
+  zoom: number = DEFAULT_ZOOM;
+  accuracy: number = -1;
+  intervalHandler: any;
+
+  constructor(private viewCtrl: ViewController, private platform: Platform) {
```

Comment:
Don't use a line skip here since in the entire project we didn't skip a line in an empty constructor. Consistency is important.